crates/uv-workspace/src/pyproject.rs

```diff
@@ -497,6 +497,36 @@ pub struct ToolUv {
     )]
     pub environments: Option<SupportedEnvironments>,
 
+    /// A list of required platforms, for packages that lack source distributions.
+    ///
+    /// When a package does not have a source distribution, it's availability will be limited to
+    /// the platforms supported by its built distributions (wheels). For example, if a package only
+    /// publishes wheels for Linux, then it won't be installable on macOS or Windows.
+    ///
+    /// By default, uv requires each package to include at least one wheel that is compatible with
+    /// the designated Python version. The `required-platforms` setting can be used to ensure that
+    /// the resulting resolution contains wheels for specific platforms, or fails if no such wheels
+    /// are available.
+    #[cfg_attr(
+        feature = "schemars",
+        schemars(
+            with = "Option<Vec<String>>",
+            description = "A list of environment markers, e.g., `sys_platform == 'darwin'."
+        )
+    )]
+    #[option(
+        default = "[]",
+        value_type = "str | list[str]",
+        example = r#"
+            # Require that the package is available for macOS ARM and x86 (Intel).
+            required-platforms = [
+                "sys_platform == 'darwin' and platform_machine == 'arm64'",
+                "sys_platform == 'darwin' and platform_machine == 'x86_64'",
+            ]
+        "#
+    )]
+    pub required_platforms: Option<SupportedEnvironments>,
```

Comment:
Should this be `required_environments` or `required_markers`? The other markers list is `environments`, `platforms` lacks symmetry

Reply:
It was somewhat intentional but I can change it.

Reply:
We might want to change the other one? I feel like `environments` is a bit overloaded in the longterm.